### PR TITLE
Add logrocket for cloud-based logging tracking

### PR DESCRIPTION
**Pull request by hiddentao**

### CODE REVIEW

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr96-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr96-app-wearekickback.surge.sh

File: scripts/setup.js [hunk 1]
```diff
@@ -26,11 +33,14 @@ if (undefined === appConfig.NUM_CONFIRMATIONS) {
...
   appConfig.API_URL = 'https://ropsten.api.kickback.events'
   appConfig.GIT_COMMIT = getGitCommit()
+  appConfig.LOGROCKET_TOKEN = '5gnafo/kickback-ropsten'
 } else if (argv.rinkeby) {
   appConfig.API_URL = 'https://rinkeby.api.kickback.events'
   appConfig.GIT_COMMIT = getGitCommit()
+  appConfig.LOGROCKET_TOKEN = '5gnafo/kickback-rinkeby'
 } else if (argv.live) {
   appConfig.MIXPANEL_ID = '11a2f7a59470cdb46cb611c5d22876f2'
+  appConfig.LOGROCKET_LIVE = '5gnafo/kickback-live'
```
Comment (makoto) [inline scripts/setup.js]: Like image uploading, can people spam the network if we expose token?

Comment (hiddentao) [reply scripts/setup.js]: Yeah, but we have this problem with our mixpanel id as well.

The difference is that we really want to control cloudinary data carefully, whereas in this case the data controllers are another company.

File: src/GlobalState.js [hunk 2]
```diff
@@ -108,6 +109,8 @@ class Provider extends Component {
   setUserProfile = profile => {
     console.log('Current user', profile)
 
+    LogRocket.identify(profile)
```
Comment (makoto) [inline src/GlobalState.js]: I think we should only send username (which is a public info) but not other user sensitive info.
We still don't know how LogRocket treat data and we don't want to pass too much data to them.

File: scripts/setup.js (hunk 1 above)
Comment (makoto) [reply scripts/setup.js]: ah, so we don't care if someone spam their network.

File: src/GlobalState.js (hunk 2 above)
Comment (hiddentao) [reply src/GlobalState.js]: Done

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr96-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr96-app-wearekickback.surge.sh

**Comment by hiddentao** [comment]
@makoto please re-review

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr96-app-wearekickback.surge.sh

**Pull request merged**